libcxx/test/libcxx/language.support/nodiscard.verify.cpp

```diff
@@ -0,0 +1,93 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03
+
+// check that <utility> functions are marked [[nodiscard]]
+
+#include <compare>
+#include <coroutine>
+#include <functional>
```

Comment:
I don't think we need to obtain `hash` from `<functional>`. `<coroutine>` already provides it.
```suggestion
```

Reply:
I though so. Thank you!